src/systems/triggered_publisher/TriggeredPublisher.cc

```diff
@@ -598,31 +612,73 @@ void TriggeredPublisher::Configure(const Entity &,
   else
   {
     ignerr << "No ouptut specified" << std::endl;
-    return;
+  }
+
+  if (sdfClone->HasElement("service"))
```

Comment:
This isn't totally of your doing, but this `Configure` function is complicated enough that I'd consider refactoring it.  In particular, what it seems to be doing is setting up the inputs, setting up the outputs, and (now) setting up the services.  I'd refactor it to be something like:

```
void TriggeredPublisher::Configure(const Entity &,
    const std::shared_ptr<const sdf::Element> &_sdf,
    EntityComponentManager &,
    EventManager &)
{
    this->configureInputs();
    this->configureOutputs();
    this->configureServices();

    this->subscribeToInput();
}
```

(I likely left out some stuff there, but you get the idea)

Reply:
This sounds good